doc/sphinx/language/gallina-extensions.rst

```diff
@@ -34,8 +34,10 @@ expressions. In this sense, the :cmd:`Record` construction allows defining
    the default name :n:`Build_@ident`, where :token:`ident` is the record name, is used. If :token:`sort` is
    omitted, the default sort is :math:`\Type`. The identifiers inside the brackets are the names of
    fields. For a given field :token:`ident`, its type is :n:`forall @binders, @type`.
-   Remark that the type of a particular identifier may depend on a previously-given identifier. Thus the
-   order of the fields is important. Finally, :token:`binders` are parameters of the record.
+   Remark that the type of a particular field may depend on a previous field. Thus the
```

Comment:
This is not about your changes but I think that "Remark that" is a Frenchism and should be replaced by "Notice that"

Reply:
Thanks for all your remarks. Now integrated.